### PR TITLE
Fight description

### PR DESCRIPTION
**Pull request by SethBailey**
this will randomize a description after you win or loose a fight

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -30,10 +30,10 @@ class TheGame
 
         public void Begin()
         {
-            playerType();
-
             if (isPlayerNew)
             {
+                playerType();
```
Comment (Stackingit) [inline Program.cs:35]: This adds no value - and was branched off the wrong branch -- you should have started the branch off of Master ( the last approved state ) not a branch that was still under review.

File: Program.cs [hunk 2]
```diff
@@ -56,6 +56,8 @@ public void Begin()
             }
             else
             {
+                playerType();
```
Comment (Stackingit) [inline Program.cs:59]: Same same

File: Program.cs [hunk 3]
```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
             return protection;
         }
 
+        private void fightDescriptionDie( Monster monster )
+        {
+            int dieDescription = new Random().Next(1,6);
+            switch ( dieDescription )
+            {
+                case 1: TypeWriter.WriteLine(new Text($"The {monster.spices} "),
+                                             new Text("stabed you through the hart ", ConsoleColor.DarkRed),
```
Comment (Stackingit) [inline Program.cs:312]: stabbed

Comment (Stackingit) [reply Program.cs:312]: heart

File: Program.cs [hunk 4]
```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
...
+                                             new Text("and danced on your grave!")); break;
+                case 2: TypeWriter.WriteLine(new Text($"the {monster.spices} "),
+                                             new Text("choped your head off ", ConsoleColor.DarkRed),
```
Comment (Stackingit) [inline Program.cs:315]: Chopped

File: Program.cs [hunk 5]
```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
...
+                                             new Text("and took it as a trophy")); break;
+                case 3: TypeWriter.WriteLine(new Text($"The {monster.spices}" ),
+                                             new Text("got you with such a mean "),
```
Comment (Stackingit) [inline Program.cs:318]: Why put this over 2 lines, you are not changing colour?

File: Program.cs [hunk 6]
```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
...
+
+        }
+
+        private void fightDescriptionWin( Monster monster )
+        {
+            
+            if (knightOrMage == "m")
+            {
+                int winDescription = new Random().Next(1,4);
+                            switch ( winDescription )
```
Comment (Stackingit) [inline Program.cs:336]: Indent issue :(

File: Program.cs [hunk 7]
```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
...
+                                                             new Text($"the {monster.spices} in a block of ice")); break;
+                            }
+            }
+            else if (knightOrMage == "k")
+            {
+                int winDescription = new Random().Next(1,4);
+                            switch ( winDescription )
+                            {
+                                case 1: TypeWriter.WriteLine(new Text("With one mighty blow you "),
+                                                             new Text("decapatated them")); break;
```
Comment (Stackingit) [inline Program.cs:355]: decapitated

Comment (Stackingit) [reply Program.cs:355]: them? ( that is plural ) ... how about "it" .. or the {monster.spieces}

File: Program.cs [hunk 8]
```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
...
+                                                             new Text("which turns them to ash")); break;
+                                case 3: TypeWriter.WriteLine(new Text("You "),
+                                                             new Text("freeze ", ConsoleColor.DarkCyan),
+                                                             new Text($"the {monster.spices} in a block of ice")); break;
+                            }
+            }
+            else if (knightOrMage == "k")
+            {
+                int winDescription = new Random().Next(1,4);
+                            switch ( winDescription )
```
Comment (Stackingit) [inline Program.cs:352]: indent :(

File: Program.cs [hunk 9]
```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
...
+                            {
+                                case 1: TypeWriter.WriteLine(new Text("With one mighty blow you "),
+                                                             new Text("decapatated them")); break;
+                                case 2: TypeWriter.WriteLine(new Text("You "),
+                                                             new Text("split them in half ", ConsoleColor.DarkCyan),
```
Comment (Stackingit) [inline Program.cs:357]: them

File: Program.cs (hunk 5 above)
Comment (SethBailey) [reply Program.cs:315]: no idea

File: Program.cs (hunk 6 above)
Comment (SethBailey) [reply Program.cs:333]: all fixed

File: Program.cs (hunk 7 above)
Comment (SethBailey) [reply Program.cs:352]: I just used {monster.spieces}

File: Program.cs (hunk 8 above)
Comment (SethBailey) [reply Program.cs:349]: same same

File: Program.cs (hunk 9 above)
Comment (SethBailey) [reply Program.cs:354]: just used the monsters name

File: Program.cs (hunk 4 above)
Comment (SethBailey) [reply Program.cs:312]: yup

File: Program.cs (hunk 3 above)
Comment (SethBailey) [reply Program.cs:309]: done

**Pull request merged**